### PR TITLE
GEODE-8247: modified FilterRoutingInfo checks

### PR DESCRIPTION
**Pull request by mivanac**
Thank you for submitting a contribution to Apache Geode.

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [*] Is there a JIRA ticket associated with this PR? Is it referenced in the commit message?

- [*] Has your PR been rebased against the latest commit within the target branch (typically `develop`)?

- [*] Is your initial contribution a single, squashed commit?

- [*] Does `gradlew build` run cleanly?

- [*] Have you written or updated unit tests to verify your changes?

- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?

### Note:
Please ensure that once the PR is submitted, check Concourse for build issues and
submit an update to your PR as soon as possible. If you need help, please send an
email to dev@geode.apache.org.

### CODE REVIEW

File: geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java [hunk 1]
```diff
@@ -196,6 +252,32 @@ public void close() {
     }
   }
 
+  private class TestCqListener2 implements CqListener, Serializable {
+    public int onEventCalls = 0;
+    public int onEventUPDATE = 0;
```
Comment (mkevo) [inline geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java:257]: Please change names, e.g. onEventCalls -> **onEventCreateCalls**, onEventUPDATE -> **onEventUpdateCalls**

File: geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java [hunk 2]
```diff
@@ -196,6 +252,32 @@ public void close() {
...
+      if (queryOperation.isCreate()) {
+        onEventCalls++;
+      } else if (queryOperation.isUpdate()) {
+        onEventUPDATE++;
+      }
+    }
+
+    @Override
+    public void onError(CqEvent aCqEvent) {
+
```
Comment (mkevo) [inline geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java:271]: remove blank line

File: geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java [hunk 3]
```diff
@@ -196,6 +252,32 @@ public void close() {
...
+    }
+
+    @Override
+    public void close() {
+
```
Comment (mkevo) [inline geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java:276]: remove blank line

Comment (mivanac) [reply geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java:276]: Thanks for comments. Updated

Comment (mkevo) [reply geode-cq/src/distributedTest/java/org/apache/geode/cache/query/cq/CQDistributedTest.java]: Thanks!

**Pull request merged**